src/packages/@ncigdc/components/analysis/ClinicalAnalysisResult.js

```diff
@@ -432,28 +486,29 @@ const ClinicalAnalysisResult = ({
               //   );
               // }
               return (
-                <ClinicalVariableCard
-                  key={varFieldName}
-                  fieldName={varFieldName}
-                  variable={varProperties}
-                  plots={plotTypes[varProperties.plotTypes || 'categorical']}
-                  style={{ minWidth: controlPanelExpanded ? 310 : 290 }}
-                  id={id}
-                  setId={setId}
-                  facetField={varFieldName.replace('cases.', '')}
-                  filters={{
-                    op: 'and',
-                    content: [
-                      {
-                        op: '=',
-                        content: {
-                          field: `cases.case_id`,
-                          value: [`set_id:${setId}`],
+                <div>
```

Comment:
remove `<div />` around `<ClinicalVariableCard />`, getting a react key error

Reply:
removed the div